### PR TITLE
Added MSP to GPS provider options in configuration tab

### PR DESCRIPTION
**Pull request by awolf78**
With reference to https://github.com/betaflight/betaflight/pull/6662

### CODE REVIEW

File: src/js/tabs/configuration.js [hunk 1]
```diff
@@ -529,7 +529,8 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
         // generate GPS
         var gpsProtocols = [
             'NMEA',
-            'UBLOX'
+            'UBLOX',
+            'MSP'
```
Comment (mikeller) [inline src/js/tabs/configuration.js]: This will need to be version gated so that it is only available for API versions 1.41 and up.

Comment (awolf78) [reply src/js/tabs/configuration.js]: How about that:
```
        var gps_protocol_e = $('select.gps_protocol');
        var gpsProtocolsLength = 2;
        if (semver.gte(CONFIG.apiVersion, "1.41.0")) {
            gpsProtocolsLength = 3;
        }
        for (var i = 0; i < gpsProtocolsLength; i++) {
            gps_protocol_e.append('<option value="' + i + '">' + gpsProtocols[i] + '</option>');
        }
```

Comment (McGiverGim) [reply src/js/tabs/configuration.js]: Usually, if version bigger than X, we append the new element to the combo. But your solution is not bad neither.

**Comment by awolf78** [comment]
Not sure why this failed, but looking at the logs it doesn't have anything to do with my code?

**Comment by mikeller** [comment]
Some times Travis just has a bad day...

**Pull request merged**